package.json

```diff
@@ -29,11 +29,20 @@
     "lint:style": "stylelint '**/*.{,s}css' --ignore-path .gitignore --max-warnings 0",
     "lint:prettier": "prettier . --check --ignore-path .gitignore",
     "test": "jest --coverage",
+    "cy": "concurrently -k yarn:e2e:serve yarn:cypress:open",
```

Comment:
Not sure I like naming `cy` - maybe something like `cypress:interactive`? I know it's longer, but makes a little more sense when reading it.

Reply:
`cypress:open`?

Reply:
lol that's already the sub script. I mean `e2e:open`?

Reply:
No that doesn't work either, it'll be confusing for `e2e` which can no longer run `npm:e2e:*`

Reply:
Perhaps simply `cypress`?

Reply:
Just to be clear, you want this as a script only to avoid having 2 terminal window open right?

But when you think about it, mostly if you run tests you'd also developing so you _would_ have Storybook running? So can it just be instruction to run `yarn cypress open` whiles running a Storybook instead of script(s)?

Reply:
Correct, single terminal, two processes.

I guess it could, this is usually a good script for test engineers who rarely open Storybook alone.

I don't mind removing `cy` and `cypress:open` if you think they're unnecessary.

Reply:
I think we could start with having command instruction on docs (the ones you just added), and see if we need to have scripts?

So we would have more time for the most difficult thing - naming :trollface:

Reply:
Sounds good.

Reply:
Oh wait, it needs to build Storybook with an environment variable, `NODE_ENV=e2e`.

Do we want the instruction to read like:

```
Run Storybook in e2e mode:
   NODE_ENV=e2e yarn start

Open Cypress:
    yarn cypress open
```

Reply:
So I suggest this:

- remove scripts `cy` and `cypress:open`
- instruct for such to serve the app with `yarn e2e:serve` and then run `yarn cypress open`

How does this sound?

Reply:
Done.